runtime/src/bank.rs

```diff
@@ -1370,11 +1370,14 @@ impl Bank {
         self.store_account_and_update_capitalization(pubkey, &new_account);
     }
 
-    fn inherit_specially_retained_account_balance(
+    fn inherit_specially_retained_account_fields(
         &self,
         old_account: &Option<AccountSharedData>,
-    ) -> u64 {
-        old_account.as_ref().map(|a| a.lamports).unwrap_or(1)
+    ) -> InheritableAccountFields {
+        (
+            old_account.as_ref().map(|a| a.lamports).unwrap_or(1),
+            INITIAL_RENT_EPOCH,
```

Comment:
... so unless this argument is reading from `old_account.rent_epoch`, it's perfectly no-functional-change.